tests/utils/test_logging_utils.py

```diff
@@ -98,6 +98,12 @@ def test_event_logging_stream_flushes_properly():
     assert stream.flush_count > 0
 
 
+def test_ignore_logging_if_stderr_is_none():
+    sys.stderr = None
+
+    eprint("foo", flush=True)
```

Comment:
Can we use `mock` here?

Reply:
+1 - missed this

Reply:
Ok, using `monkeypatch` although there is already a test fixture called `reset_stderr` which restores `sys.stderr` after each test.